### PR TITLE
docs/podman.1: Link to hook documentation

### PR DESCRIPTION
**Pull request by wking**
Collecting the Podman/libpod specifics in one place allows us to make the hooks README more generic.

I think this is a prereq to adding a Podman/libpod extended stage using the API from #758.

I've currently used a Markdown link here, although links aren't really a man-page thing.  The link will work on GitHub, but we may want some pre-processing to replace the link (and [our other existing links][1]?) when building for installation.  Maybe that means translating the current hooks README into a man page as well.  `oci-hooks.5`?

[1]: https://github.com/projectatomic/libpod/blame/v0.5.2/docs/podman.1.md#L60

### CODE REVIEW

File: docs/podman.1.md [hunk 1]
```diff
@@ -121,6 +121,14 @@ has the capability to debug pods/images created by crio.
 
 	Note this is not a volume mount. The content of the volumes is copied into container storage, not bind mounted directly from the host.
 
+**hook JSON** (`/usr/share/containers/oci/hooks.d/*.json`)
+
+  Each `*.json` file in `/usr/share/containers/oci/hooks.d` configures a hook for Podman containers.  For more details on the syntax of the JSON files and the semantics of hook injection, see [the hook documentation](../pkg/hooks/README.md).
```
[the remark this answers is not in the document]
Comment (wking) [reply docs/podman.1.md]: > I don't think relative links in a manpage will work...

Thoughts on the ideas I floated in the final paragraph of the topic post?

[the remark this answers is not in the document]
Comment (mheon) [reply docs/podman.1.md]: I'd be fine with making the hooks README a manpage and referencing it from this manpage

**Comment by wking** [comment]
I've pushed 57a1651 -> 0f0954a, converting the hooks docs to `oci-hooks.5`.  The new series builds on #774, so review that first.

https://github.com/projectatomic/libpod/pull/772#discussion_r188341120

**Comment by rhatdan** [comment]
@wking Needs rebase.

**Comment by wking** [comment]
Rebased onto master with d2b931a -> 40a97ac (no conflicts).

**Comment by mheon** [comment]
@wking This needs a rebase

**Pull request closed without merging**